bcrs-ui/public/config/configuration.json

```diff
@@ -0,0 +1,6 @@
+{
+  "API_URL": "https://legal-api-dev.pathfinder.gov.bc.ca/api/v1/businesses/",
+  "AUTH_API_URL": "https://auth-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "PAY_API_URL": "https://pay-api-dev.pathfinder.gov.bc.ca/api/v1/",
+  "ADDRESS_COMPLETE_KEY": "RH99-RN99-GB72-FW86"
+}
```

Comment:
Please add latest "KEYCLOAK_CONFIG_URL" key and value, and also its related code (in config-helper.ts, main.ts and vue.config.js).

Reply:
oh nice, some new stuff! I didn't notice the changes to master, i will get this work included. Thanks for the heads up :)

Reply:
I guess this leads to another question, since this work directly correlates with Authentication, are we thinking to pull that in as well?

Reply:
Yes, this authorization code is needed in this project.

I also this this code should be included in the UI template. For the foreseeable future, this is the authorization code that we will use in UI projects. And the UI template can easily be updated later as needed.

Reply:
ok perfect, we will get it all in there, thanks!

Reply:
You need one last thing: the `devServer` config in vue.config.js so that, when running locally, the subject keycloak.json file is retrieved from the deployed auth project (so we don't need a local copy of it).

Reply:
K great idea. Will do!